sdk/cosmos/azure-cosmos/README.md

```diff
@@ -681,10 +681,30 @@ indexing_policy = {
         ],
         "vectorIndexes": [
             {"path": "/vector1", "type": "flat"},
-            {"path": "/vector2", "type": "quantizedFlat"}
+            {"path": "/vector2", "type": "quantizedFlat"},
+            {"path": "/vector2", "type": "diskANN"}
         ]
     }
 ```
+
+For vector index types of diskANN and quantizedFlat, there are additional options available as well. These are:
+
+quantizationByteSize - the number of bytes used in product quantization of the vectors. A larger value may result in better recall for vector searches at the expense of latency. This applies to index types diskANN and quantizedFlat. The allowed range is between 1 and 256, and the default value is 64.
+
+vectorIndexShardKey - the list of string containing the shard keys used for partitioning the vector indexes. This applies to index types diskANN and quantizedFlat.
```

Comment:
Please add a comment indicating that the maximum allowed length of this array currently is 1 i.e. user's can only provide a single path